docs/user-guide/migrating-to-3.0.0.md

```diff
@@ -0,0 +1,79 @@
+# Migrating to v3.0.0
+
+ESLint v3.0.0 is the third major version release. We have made several breaking changes in this release, however, we believe the changes to be small enough that they should not require significant changes for ESLint users. This guide is intended to walk you through the changes.
+
+## Dropping Support for Node.js < 4
+
+With ESLint v3.0.0, we are dropping support for Node.js versions prior to 4. Node.js 0.10 and 0.12 are in [maintenance mode](https://github.com/nodejs/LTS) and Node.js 4 is the current LTS version. If you are using an older version of Node.js, we recommend upgrading to at least Node.js 4 as soon as possible. If you are unable to upgrade to Node.js 4 or higher, then we recommend continuing to use ESLint v2.x until you are ready to upgrade Node.js.
+
+**Important:** We will not be updating the ESLint v2.x versions going forward. All bug fixes and enhancements will land in ESLint v3.x.
+
+## Requiring Configuration to Run
+
+ESLint v3.0.0 now requires that you use a configuration to run. A configuration can be any of the following:
+
+1. A `.eslintrc.js`, `.eslintrc.json`, `.eslintrc.yml`, `.eslintrc.yaml`, or `.eslintrc` file either in your project or home directory.
+2. Configuration options passed on the command line using `--rule`, `--global`, or `--env`.
+3. A configuration file passed on the command line using `-c`.
+4. A configuration passed to `CLIEngine` with the `useSpecificConfig` option.
+
+If ESLint can't find a configuration, then it will throw an error and ask you to provide one.
+
+This change was made to help new ESLint users who are frequently confused that ESLint does nothing by default besides reporting parser errors. We anticipate this change will have minimal impact on most established users because you're more likely to have configuration files already.
+
+**To Address:** You should be sure to use a configuration whenever you run ESLint. However, you can still run ESLint without a configuration by passing the `--no-eslintrc` option on the command line or setting the `useEslintrc` option to `false` for `CLIEngine`.
+
+To create a new configuration, use `eslint --init`.
+
+## Changes to `"eslint:recommended"`
+
+```json
+{
+    "extends": "eslint:recommended"
+}
+```
+
+In 3.0.0, the following rules were added to `"eslint:recommended"`.
+
+* [`no-unsafe-finally`](http://eslint.org/docs/rules/no-unsafe-finally) helps catch `finally` clauses that may not behave as you think.
+* [`no-native-reassign`](http://eslint.org/docs/rules/no-native-reassign) was previously part of `no-undef`, but was split out because it didn't make sense as part of another rule. The `no-native-reassign` rule warns whenever you try to overwrite a read-only global variable.
+
+The following rules were removed:
```

Comment:
Although redundant, might be worth the extra words to be explicit in case anyone reads too quickly:

The following rules were removed from `"eslint": "recommended"`